index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Global Travel Agency</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="top-navigation">
+        <ul class="top-navigation-list">
+          <li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+          <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+          <li><a class="top-nav-text" href="#">English</a></li>
+          <li><a class="top-nav-text" href="#">My Page</a></li>
+          <li><a class="top-nav-text" href="#">Logout</a></li>
```

Comment:
- [ ] Check your design layout, the original one top nav ends where the News ends

Original design:

![image](https://user-images.githubusercontent.com/60631456/124144233-65335080-da51-11eb-9bc1-e335c03a13fa.png)

Your design:

![image](https://user-images.githubusercontent.com/60631456/124144336-7bd9a780-da51-11eb-8653-78956ae3ddf2.png)

It looks worse in my screen because I have a widescreen but if you fix this, it doesn't matter the size of the screen.

Reply:
It is now fixed